pep-0554.rst

```diff
@@ -64,128 +64,127 @@ the `"interpreters" Module API`_ section below.
 
 For creating and using interpreters:
 
-+------------------------------+----------------------------------------------+
-| signature                    | description                                  |
-+==============================+==============================================+
-| list_all() -> [Intepreter]   | Get all existing interpreters.               |
-+------------------------------+----------------------------------------------+
-| get_current() -> Interpreter | Get the currently running interpreter.       |
-+------------------------------+----------------------------------------------+
-| create() -> Interpreter      | Initialize a new (idle) Python interpreter.  |
-+------------------------------+----------------------------------------------+
++----------------------------------+----------------------------------------------+
+| signature                        | description                                  |
++==================================+==============================================+
+| ``list_all() -> [Intepreter]``   | Get all existing interpreters.               |
```

Comment:
"Interpreter" is misspelled here

Reply:
gah, I didn't see this until after I merged (hadn't reloaded the page).  I'll fix in a separate PR.